src/main/resources/languages/lv/messages.yml

```diff
@@ -35,7 +35,7 @@ guide:
     languages: 'Izvēlies savu vēlamo valodu,'
     credits: 'Slimefun4 Ieguldītāji'
   credits:
-    commit: "Ieguldījums"
+    commit: "\\\\t\nIeguldījums"
```

Comment:
what's going on here :thinking:

Reply:
gitlocalize had a lot of issues with tab and new line characters.
Luckily, Crowdin detects them, so we we can fix it